.gitpod.yml

```diff
@@ -0,0 +1,21 @@
+image:
+  file: .gitpod.Dockerfile
```

Comment:
Just to keep this a little cleaner, could we move this into `.gitpod/Dockerfile` (as a directory) and then move the Dockerfile?